Makefile.am

```diff
@@ -9,15 +9,14 @@ BUILT_FILES =
 ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
 AM_MAKEFLAGS = --no-print-directory
 
-
-
 GCC_COLORS ?= 'yes'
 export GCC_COLORS
 
 BUILT_SOURCES = include/mavlink/ardupilotmega/mavlink.h
 
 clean-local:
 	rm -rf $(top_builddir)/include/mavlink
+	rm -rf $(EXTRA_PROGRAMS)
```

Comment:
this should not be needed if you don't call make again but rather let them be simple targets